src/scripts/run-tests.sh

```diff
@@ -39,7 +39,11 @@ fi
     'HTMLModelCoverage','${PARAM_MODEL_COVERAGE_HTML}',\
     'SimulinkTestResults','${PARAM_TEST_RESULTS_SIMULINK_TEST}',\
     'HTMLTestReport','${PARAM_TEST_RESULTS_HTML}',\
-    'PDFTestReport','${PARAM_TEST_RESULTS_PDF}');\
+    'PDFTestReport','${PARAM_TEST_RESULTS_PDF}',\
+    'Strict',${PARAM_STRICT},\
+    'UseParallel',${PARAM_USE_PARALLEL},\
```

Comment:
I just noticed that Strict and UseParallel options don't have single quotes around the parameter. I think this is because they're both boolean inputs, but I was wondering we would see an error if the user put `False` (capital F) or `no` as the inputs because both of them are valid yaml but not matlab boolean.

Can you confirm to verify the behavior? I wonder if we might need to translate it to `false`.

Reply:
Yes, the quotes are absent as these are Boolean inputs. For both `False` and `no`, the script is generated i.e. the task doesn't error out. But also there isn't any statement in the logs which would confirm if these were applied as boolean `false`.

Reply:
It looks like the tests still passed when you had `True` input so it seems like it's working as expected

Reply:
Yes, to be sure I tried with `True` and it worked so that's good news!